torch/onnx/utils.py

```diff
@@ -90,6 +90,10 @@ def export(model, args, f, export_params=True, verbose=False, training=False,
             OperatorExportTypes.ONNX_ATEN_FALLBACK: if symbolic is missing,
                                                     fall back on ATen op.
             OperatorExportTypes.RAW: export raw ir.
+        opset_version (int, default is 9): by default we export the model to the
```

Comment:
Then each time we bump our by default opset_version, we will need to bump here as well.

Reply:
Default is None. :-) So it will automatically bumped with _default_onnx_opset_version, but, yes, the comment needs to be updated... probably I will use _default_onnx_opset_version to replace it in the next PR. :-)